pkg/runner/testdata/issue-1195/push.yml

```diff
@@ -0,0 +1,13 @@
+on: push
+
+env:
+  variable: "${{ github.repository_owner }}"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: print env.variable
+        run: |
+          echo ${{ env.variable }}
+          exit ${{ (env.variable == 'nektos') && '0' || '1'}}
```

Comment:
This test will fail in forks of act outside of the nektos org, maybe use a different context value like `github.event_name`. I still approve this change as is.